src/app/Main/Body/QueryPage/Results/TableResults/TableContent-Header.component.tsx

```diff
@@ -24,22 +40,12 @@ const StyledTableCell = withStyles(() =>
 
 export interface TableContentHeaderProps {
     headerValues: string[];
-    getSize: (
-        node: HTMLElement | null,
-        key: "height" | "totalWidth" | "partWidth"
-    ) => void;
-    isRowNotCol: boolean;
     isRowAndCol: boolean;
-    style: (isRow: boolean, isRowAndCol: boolean) => SerializedStyles;
 }
 
 /**
  * @desc Returns list of table cells for the table header
  * @param {string[]} headerValues - object with two booleans, true if row/column is selected
- * @param {(node: HTMLElement | null, key: "height" | "totalWidth" | "partWidth") => void} getSize - callback function to get the size of the header for the position of the main header
- * @param {boolean} isRowNotCol - true if row and no column is selected
- * @param {boolean} isRowAndCol - true if row and column is selected
- * @param {(isRow: boolean, isRowAndCol: boolean) => SerializedStyles} - style of the header

```

Comment:
you should move the param comments to the props interface description which is the actual input of the TableContentHeaderComponent

Reply:
Done.